packages/transaction-controller/src/types.ts

```diff
@@ -100,6 +102,76 @@ export interface Transaction {
   estimateGasError?: string;
 }
 
+/**
+ * @type TransactionReceipt
+ *
+ * Following properties are not included in the type definition but are returned by the API:
+ * @property byzantium - True if the block is in a post-Byzantium Hard Fork block.
```

Comment:
If these are consistently returned and we're documenting them anyway, is it cleaner to just include them as standard properties with inline descriptions?

Reply:
Definitely agree, I think it adds too much ambiguity if we have two sets of values and descriptions.

Shall we just remove these JSDoc properties and just stick with the inline properties and descriptions we have?